### PR TITLE
Document schema changes with DoctrineMigrationsBundle

### PR DESCRIPTION
**Pull request by cedricziel**
With DoctrineMigrationsBundle, the schema changes will be loaded transparently through a schema listener so that users can apply the schema changes for AclBundle by simply diffing the schema.

### CODE REVIEW

File: src/Resources/doc/index.rst [hunk 1]
```diff
@@ -95,6 +95,19 @@ running the following command:
 .. code-block:: bash
 
     $ php bin/console acl:init
+    
+If you are using `DoctrineMigrationsBundle`, the schema changes can be applied
```
Comment (xabbuh) [inline src/Resources/doc/index.rst]: missing underscore after the link text:

```rst
`DoctrineMigrationsBundle`_
```

File: src/Resources/doc/index.rst [hunk 2]
```diff
@@ -95,6 +95,19 @@ running the following command:
...
+by diffing your current schema.
+
+.. code-block:: bash
```
Comment (xabbuh) [inline src/Resources/doc/index.rst]: The Symfony docs use the `terminal` code block. I suggest to do the same here.

**Comment by cedricziel** [comment]
@xabbuh I updated the PR. I replaced all bash blocks with terminal ones so it's consistent.

**Comment by cedricziel** [comment]
Anything I could do to make this compliant?

**Comment by johnpancoast** [comment]
I'm new to this specific project, but I happened to come across this PR and [this part about updating with doctrine migrations](https://github.com/symfony/acl-bundle/pull/12/files#diff-83ad34138e981de744f2667057885e40R99) which seems like an important thing to clarify.

Looking at the above failed build, it seemed it just timed out on a composer update, but I'm not seeing the details on first glance.

Seems as though we should resolve whatever that issue is, merge this PR, and possibly create a `1.0.1` tag with the updated documentation for future users.

Thoughts?

**Review by oleg-andreyev — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @cedricziel.

**Pull request merged**